cli/permissions.rs

```diff
@@ -349,12 +349,10 @@ fn permission_prompt(_message: &str) -> bool {
 }
 
 fn log_perm_access(message: &str) {
-  if log_enabled!(log::Level::Info) {
-    eprintln!(
-      "{}",
-      colors::bold(format!("{}️  Granted {}", PERMISSION_EMOJI, message))
-    );
-  }
+  debug!(
+    "{}",
+    colors::bold(format!("{}️  Granted {}", PERMISSION_EMOJI, message))
+  );
```

Comment:
With changing the default log level to Info, this message was now always visible and broke at least one test. (The test is easy to fix, but the spam remains.) Changing this message to debug removes the verbosity, but now one has to enable debug to print this message, which brings in a lot other messages as well. Any thoughts?